libnavigation-base/src/main/java/com/mapbox/navigation/base/options/RerouteOptions.kt

```diff
@@ -0,0 +1,22 @@
+package com.mapbox.navigation.base.options
+
+import androidx.annotation.IntRange
+
+class RerouteOptions private constructor(
+    val avoidManeuverTime: Int
```

Comment:
Shouldn't the same be added to `routeAlternativesOptions`? Or will Nav Native's alternative engine use options from the primary route (which might be better anyway)?

Reply:
Gotcha, could you quickly follow up by adding the same `avoidManeuverTime` to SDK's `RouteAlternativesOptions` and wire it to native `minTimeBeforeManeuverSeconds`?

Reply:
cut a ticket in `navigation-sdks`